lib/tapioca/rbi/rewriters/sort_nodes.rb

```diff
@@ -25,29 +25,29 @@ def visit(node)
         sig { params(node: Node).returns(Integer) }
         def node_rank(node)
           case node
-          when Group                then kind_rank(node.kind)
-          when Include, Extend      then 0
-          when Helper               then 1
-          when TypeMember           then 2
-          when MixesInClassMethods  then 3
-          when TStructField         then 4
-          when TEnumBlock           then 5
+          when Group                then group_rank(node.kind)
+          when Include, Extend      then 10
```

Comment:
Using 10, 20, etc. so we can add some sub-ordering when needed (example with Method a few lines below)